addons/web_editor/static/src/js/rte.summernote.js

```diff
@@ -398,6 +398,9 @@ eventHandler.modules.imageDialog.showImageDialog = function ($editable) {
     if (r.sc.tagName && r.sc.childNodes.length) {
         r.sc = r.sc.childNodes[r.so];
     }
+    if (r.sc.parentNode.parentNode.className.match(/(^|\s)media_iframe_video($|\s)/)) {
+        r.sc = r.sc.parentNode.parentNode;
+    }
```

Comment:
Instead of this, I think you can just change the line below by:
```javascript
new widgets.MediaDialog(null, {}, $editable, dom.isImg(r.sc) ? r.sc : $(r.sc).parents(dom.isImg)[0]).open();
```

Reply:
```javascript
new widgets.MediaDialog(null, {}, $editable, $(r.sc).parents().addBack().filter(function (i, el) {
    return dom.isImg(el);
})[0]).open();
```